warehouse/ingest-core/src/main/java/datawave/ingest/OptionsParser.java

```diff
@@ -0,0 +1,52 @@
+package datawave.ingest;
+
+import datawave.ingest.config.TableConfigCache;
+import datawave.ingest.data.config.ingest.AccumuloHelper;
+import datawave.ingest.util.ConfigurationFileHelper;
+import datawave.util.cli.PasswordConverter;
+import org.apache.hadoop.conf.Configuration;
+
+public class OptionsParser {
+    
+    // todo: we parse these same arguments across several different classes in slightly different ways with very similar flags (e.g. zookeepers, zooKeepers,
```

Comment:
is this really still a "TODO"?

Reply:
there are other options still to add, but to minimize scope creep, I've left the todo